app/controllers/dogs_controller.rb

```diff
@@ -8,8 +8,12 @@ def index
   def create
     # Create a new dog
     dog = Dog.create(dog_params)
+    if dog.valid?
     render json: dog
+    else
+      render json: dog.errors, status:422
   end
+end
 
```

Comment:
The update and destroy controller methods (Lines 25 -29) should be above the private/strong params.

Reply:
I made the required changes. Can you approve ?